README.Rmd

```diff
@@ -72,79 +83,69 @@ R version 3.6.1 and R packages:
 - caTools
 - tibble
 - purrr
-- devtools
 - pROC
 - plotROC
 - class
 - ggplot2
 - rpart
 - randomForest
+- testthat
 - datasets
 
 
-## Installation
-
-You can install the released version of RMLViz from [CRAN](https://cran.r-project.org/) with:
-
-```
-install.packages("RMLViz")
-```
-
-And the development version from [GitHub](https://github.com/) with:
-
-```
-# install.packages("devtools")
-devtools::install_github("UBC-MDS/RMLViz")
-```
-
 ## Usage and example
 
 1. `model_comparison_table`
 
-```
+``` {r 1, message=FALSE}
 library(RMLViz)
 library(mlbench)
 data(Sonar)
 
-toy_classification_data <- select(as_tibble(Sonar), V1, V2, V3, V4, V5, Class)
+toy_classification_data <- dplyr::select(dplyr::as_tibble(Sonar), V1, V2, V3, V4, V5, Class)
 
-train_ind <- createDataPartition(toy_classification_data$Class, p=0.9, list=F)
+train_ind <- caret::createDataPartition(toy_classification_data$Class, p=0.9, list=F)
 
 train_set_cf <- toy_classification_data[train_ind, ]
 test_set_cf <- toy_classification_data[-train_ind, ]
 
 ## classification models setup
-gbm <- train(Class~., train_set_cf, method="gbm", verbose=F)
-lm_cf <- train(Class~., train_set_cf, method="LogitBoost", verbose=F)
+gbm <- caret::train(Class~., train_set_cf, method="gbm", verbose=F)
+lm_cf <- caret::train(Class~., train_set_cf, method="LogitBoost", verbose=F)
```

Comment:
Don't think it's necessary to mention caret:: since I think the package will be imported with the dependencies. But we can keep it to be safe